src/credentials/attestation_verifier/DeviceAttestationVerifier.h

```diff
@@ -386,6 +393,8 @@ class DeviceAttestationVerifier
                                                            const Crypto::P256PublicKey & dacPublicKey,
                                                            const ByteSpan & csrNonce) = 0;
 
+    virtual RevocationReason IsCertificateRevoked(bool isPaa, ByteSpan issuer, ByteSpan authorityKeyId, ByteSpan serialNumber) = 0;
```

Comment:
This requires a synchronous response from the implementation.  Should this be async, similar to VerifyAttestationInformation?  Otherwise it basically requires an in-memory revocation database in the current process, right?

Reply:
This has to be async, yes